src/css/components/panel.scss

```diff
@@ -157,3 +156,23 @@ h4 + .panel {
   font-weight: 600;
   padding-bottom: $grid-1;
 }
+
+.panel-routes,
```

Comment:
This code makes sense and works, but I'm worried we have way too much in the panel component. I also am unsure we should be writing classes as specific to one use case, like `panel-services`.

I guess the question to ask is, why are we modifying these things in our heading tags just for panel services and how could we use classes and styles that already exist (there's not an easy answer to this). For instance, where does the 0.75rem style come from, is there not another `h` tag we can use for that?